app/views/admin/poll/questions/_form.html.erb

```diff
@@ -1,18 +1,21 @@
-<%= form_for(@question, url: form_url) do |f| %>
+<%= render "admin/shared/globalize_locales", resource: @question %>
+
+<%= translatable_form_for(@question, url: form_url) do |f| %>
 
   <%= render 'shared/errors', resource: @question %>
 
   <%= f.hidden_field :proposal_id %>
 
   <div class="small-12">
     <div class="small-12 medium-6 large-4">
+      <% select_options = Poll.all.map { |p| [p.name, p.id] } %>
```

Comment:
We will use `Poll.all.map...` instead of `Poll.pluck` cause we want to make use of [the fallbacks set for i18n](https://github.com/consul/consul/blob/master/config/application.rb#L24) for missing translations